doc/user_doc/vic_app_dev/using_volumes_with_vic.md

```diff
@@ -0,0 +1,66 @@
+
+<pre>[...]
+Storage Driver: vSphere Integrated Containers Backend Engine
+VolumeStores: <i>volume_store_1</i> <i>volume_store_2</i> ... <i>volume_store_n</i>
+vSphere Integrated Containers Backend Engine: RUNNING
+[...]</pre>
+
+## Create a Volume in a Volume Store ##
+
+When you use the `docker volume create` command to create a volume, you can optionally provide a name for the volume by specifying the `--name` option. If you do not specify `--name`, `docker volume create` assigns a random UUID to the volume.
+
+If the volume store label is anything other than `default`, you must specify the `--opt VolumeStore` option and pass the name of an existing volume store to it. If you do not specify `--opt VolumeStore`, `docker volume create` searches for a volume store named `default`, and returns an error if no such volume store exists. 
+
+<pre>docker -H <i>virtual_container_host_address</i>:2376 --tls volume create 
+--opt VolumeStore=<i>volume_store_label</i> 
+--name <i>volume_name</i></pre>
+
+ If you or the vSphere Administrator set the volume store label to `default` when running `vic-machine create`, you do not need to specify `--opt VolumeStore`.
+
+<pre>docker -H <i>virtual_container_host_address</i>:2376 --tls volume create 
+--name <i>volume_name</i></pre>
+
+You can optionally set the capacity of a volume by specifying the `--opt Capacity` option when you run `docker volume create`. If you do not specify the `--opt Capacity` option, the volume is created with the default capacity of 1024MB. 
+
+If you do not specify a unit for the capacity, the volume is created with a capacity in megabytes.
+<pre>docker -H <i>virtual_container_host_address</i>:2376 --tls volume create 
+--opt VolumeStore=<i>volume_store_label</i> 
+--opt Capacity=2048
+--name <i>volume_name</i></pre>
+To create a volume with a capacity in kilobytes, gigabytes, or terabytes, include `KB`, `GB`, or `TB` in the value that you pass to `--opt Capacity`. The unit is case insensitive.
+
+<pre>docker -H <i>virtual_container_host_address</i>:2376 --tls volume create 
+--opt VolumeStore=<i>volume_store_label</i> 
+--opt Capacity=10GB
+--name <i>volume_name</i></pre>
+
+
+**NOTE**: When using a vSphere Integrated Containers virtual container host as your Docker endpoint, the storage driver is always the vSphere Integrated Containers Backend Engine. If you specify the `docker volume create --driver` option, it is ignored.  
+
+## Obtain the List of Available Volumes ##
+
+To obtain a list of volumes that are available on a virtual container host, run `docker volume ls`.
+
+<pre>docker -H <i>virtual_container_host_address</i>:2376 --tls volume ls
+
+DRIVER         VOLUME NAME
+vsphere        <i>volume_1</i>
+vsphere        <i>volume_2</i>
+[...]          [...]
+vsphere        <i>volume_n</i></pre>
+
+## Delete a Named Volume from a Volume Store ##
+To delete a volume, run `docker volume rm` and specify the name of the volume to delete.
```

Comment:
This is currently not supported. The work in the docker personality and portlayer handlers is complete. But the back end is not quite there yet.